androidComposePrototype/src/main/java/com/genesys/cloud/messenger/androidcomposeprototype/ui/testbed/TestBedFragment.kt

```diff
@@ -9,12 +9,18 @@ import androidx.browser.customtabs.CustomTabsIntent
 import androidx.compose.ui.platform.ComposeView
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.activityViewModels
+import com.genesys.cloud.messenger.androidcomposeprototype.util.getSharedPreferences
 import kotlinx.coroutines.runBlocking
 
 class TestBedFragment : Fragment() {
 
     private val viewModel: TestBedViewModel by activityViewModels()
     private val onOktaSignIn: (url: String) -> Unit = { url -> launchCustomTabs(url) }
+    private val onOktaLogout: () -> Unit = {
+        context?.getSharedPreferences()?.edit()?.run {
+            remove("authCode")
+        }
```

Comment:
I think you also need a `commit()` or `apply()` here.

Reply:
> I think you also need a `commit()` or `apply()` here.

Good point there. How did I even missed that one...